File: project/Python/LinkedStacker.py
class LinkedStacker:
    top = Node(None)

    def __init__(self):
        top = None

    def isEmpty(self):
        if top is None:
            return True
        else
            return False
    
    def push(self, data):
        newNode = Node(data)
        newNode.nextNode = top
        top = newNode
        return True

    def pop(self):
        if isEmpty() == True:
            return False

        nodeToDelete = top
        top = top.nextNode
        nodeToDelete = None
        return True

    def peek(self):
        return top.data

File: project/Dart/LinkedStacker.dart
import 'Node.dart';

class LinkedStacker<V> { 
    Node<V> top;      
    
    LinkedStacker()  
    {
        top = null;
    }

    bool isEmpty() {
        if (top == null) {
            return true;
        }
        else {
            return false;
        }
    }

    bool push(V data) {
        Node<V> newNode = new Node<V>(data);
        newNode.nextNode = top;
        top = newNode;
        return true;
    }

    bool pop() {
        if (isEmpty() == true) {
            return false;
        }

        Node<V> nodeToDelete = top;
        top = top.nextNode;
        nodeToDelete = null;
        return true;
    }

    V peek() {
        return top.data;
    }
}